cmd/authelia-scripts/cmd_unittest.go

```diff
@@ -17,7 +17,7 @@ func RunUnitTest(cobraCmd *cobra.Command, args []string) {
 		log.Fatal(err)
 	}
 
-	cmd := utils.Shell("yarn test")
+	cmd := utils.Shell("npm test")
```

Comment:
Also wondering why this was changed?

Reply:
> Also wondering why this was changed?

I changed it just for me and forgot to re-set it to 'yarn' before pushing

Reply:
Gotcha!